website/static/js/comment.js

```diff
@@ -91,6 +100,21 @@ var BaseComment = function() {
         return self.submittingReply() ? 'Commenting' : 'Comment';
     });
 
+    self.updateViewportDimensions = ko.computed(function () {
+        var commentSidebarRef = $('.cp-sidebar-content');
+        var commentRef = $('.comment-body').first();
+
+        var commentsWidth = self.atBottomOfScroll() ? commentSidebarRef.width() : 300;
```

Comment:
Can you replace the hard-coded values with variables? That will make it easier to tell where the numbers came from!
e.g. `var COMMENT_SIDEBAR_DEFAULT_WIDTH = $('.cp-sidebar-content').width();`